statsmodels/compat/python.py

```diff
@@ -19,6 +26,8 @@
     "lrange",
     "lfilter",
     "with_metaclass",
+    PY37,
```

Comment:
Should these be surrounded by `""`

Reply:
I don't think we support PY37 anymore officially, could remove

Reply:
Good plan – addressed in 8ed424e02717fb1263eee2973c6c57c1ba2a50fd and b6353804fd71c8d1adc60b1dd83197d974e6fb0b.

Reply:
and I wrapped `PYTHON_IMPL_WASM` in `__all__` in quotes in 98005e1f55a561bb3ace51a0b0f251f55d25b71b.